src/tutorials/learn-how-to-deploy-with-truffle-teams.md

```diff
@@ -0,0 +1,122 @@
+
+![Commit Card](/img/tutorials/learn-how-to-deploy-with-truffle-teams/commit-card.png)
+
+### Staging and Production Columns
+
+**Staging** contains a list of all testnet (i.e. Ropsten, Görli, Rinkeby, and Kovan) deployments. **Production** contains a list of Mainnet deployments. We'll expand more on how these work later in the tutorial.
+
+## Your First Deployment
+
+Currently, Truffle Teams supports Ethereum testnets and Mainnet, and we'll be adding support for other targets over time.
+
+In this tutorial, we'll be deploying our application to Ropsten.
+
+Press the parachute icon on the commit you'd like to deploy.
+
+![Parachute Callout](/img/tutorials/learn-how-to-deploy-with-truffle-teams/parachute-callout.png)
+
+A wizard will pop up; under Destination Network, select Ropsten, and then press **Connect Wallet**.
+
+![Deployment Wizard Ropsten Select](/img/tutorials/learn-how-to-deploy-with-truffle-teams/deployment-wizard-ropsten-select.png)
+
+MetaMask will pop up asking you to login (if you haven't already). Then you will need to press **Connect** to confirm the connection with Truffle Teams.
+
+![MetaMask Connect](/img/tutorials/learn-how-to-deploy-with-truffle-teams/metamask-connect.png)
+
+If the wizard hasn't changed (i.e. you don't see a button to start deploying), you likely need to switch your MetaMask network to Ropsten. This may reload the page; if this happens, you'll need to repeat the steps above.
+
+![MetaMask Network](/img/tutorials/learn-how-to-deploy-with-truffle-teams/metamask-network.png)
+
+Before moving on, make sure you have the correct account selected in MetaMask.
+
+![MetaMask Account](/img/tutorials/learn-how-to-deploy-with-truffle-teams/metamask-account.png)
+
+Now you should see the wizard prompting for a **Deployment Context**. Leave it set to **Create a New Deployment**. We'll cover what this is later in the tutorial. Press the **OK, START DEPLOYING** button!
+
+![Deployment Wizard Ready](/img/tutorials/learn-how-to-deploy-with-truffle-teams/deployment-wizard-ready.png)
+
+From here on, it's important that you don't close the tab, refresh the page, or lose network connection. We're working on a more robust experience that will enable you to pick up unfinished deployments, but for now our current version requires the tab to stay open and connected.
+
+Truffle Teams can only process so many deployments at the same time, so you may see that your deployment has been queued. You'll have to wait (without closing/refreshing the tab) for your deployment to get to the front of the list. However, we're working diligently to make this experience more seamless going forward.
```

Comment:
I would maybe change 'can' to 'will'.  This way we don't give the impression that we "can't".  Maybe we can even lead with "During the beta period, etc.."
^ just a thought